pydrive2/test/test_oauth.py

```diff
@@ -82,7 +82,6 @@ def test_06_ServiceAuthFromSavedCredentialsP12File(self):
         time.sleep(1)
 
     def test_07_ServiceAuthFromSavedCredentialsJsonFile(self):
-        setup_credentials("credentials/7.dat")
```

Comment:
The whole point of this test is (at least that was what I inferred from its name) using the saved credentials. So I changed the `saved_credentials_file` to `credentials/default.dat` which would contain the saved credentials from the first `setup_credentials` call in the `setup_class` and do the test

Reply:
I think `credentials` is a misleading name here. If we take the name of the existing test `test_06_ServiceAuthFromSavedCredentialsP12File` it probably means that it should be testing a scenario when we P12 and we can auth using it. The same should be with Json service account file. The same is true for all other auth tests in this file - they all delete saved credentials and proceed with actual Auth flow - I guess that's the whole point of it - test all different possible options to Auth.

As far as I can tell (not going to deep, so I might miss something) the previous P12 test was bad and was not testing the right thing. As well the JSON one that we have now.

Reply:
@shcheklein I tried to ensure by logging in twice in that test, would you check out the latest version

Reply:
@isidentical 

> I tried to ensure by logging in twice in that test, would you check out the latest version

could you please clarify, what specifically you were trying to check/test by logging?

Reply:
s/logging/logining in twice/. The first login ensures that the credentials gets saved up so that the second call is the actual test which authenticates with the saved credentials (as the test name implies)

Reply:
gotcha! it makes total sense. I can bet it will be confusing still for people that we mix JSON credentials file in this test with actual saved credentials in other tests. I would put at least a comment about this if possible, that in this case `setup_credentials` that is called with the default value actually writes the JSON service file.